modules/@apostrophecms/modal/ui/apos/mixins/AposDocsManagerMixin.js

```diff
@@ -5,6 +5,7 @@ import { detectDocChange } from 'Modules/@apostrophecms/schema/lib/detectChange'
 // Vue 3 using the composition API. - AB
 
 import klona from 'klona';
+import _ from 'lodash';
```

Comment:
Use `Array.from(new Set(array))` to unique-ify an array without hauling in lodash. isObject can be `(typeof foo === 'object')`, isString can be `(typeof foo === 'string')`.